httr2/httr2_examples.R

```diff
@@ -15,15 +15,15 @@ endpoint_openai <- "/images/generations"
 endpoint_gitlab <- "/projects/53405032/issues"
 
 # Define request
-req_openai <- request(paste0(base_url_openai, endpoint_openai))
-req_gitlab <- request(paste0(base_url_gitlab, endpoint_gitlab))
+req_openai <- request(base_url_openai) %>% 
+  req_url_path_append(endpoint_openai)
+req_gitlab <- request(base_url_gitlab) %>% 
+  req_url_path_append(endpoint_gitlab)
 
 # Provide access token via header
 headers_openai <- req_openai %>%
-  req_headers(
-    `Content-Type` = "application/json",
-    `Authorization` = paste0("Bearer ", access_token_openai)
-  )
+  req_auth_bearer(access_token_openai)
```

Comment:
There happens to be a custom function specifically for bearer tokens, but there's some advantage for doing it by hand if you're teaching folks.

You don't need the content type header because `req_body_json()` adds it for you automatically.